doc/source/release-notes/Percona-Server-8.0.16-7.rst

```diff
@@ -0,0 +1,103 @@
+series. All of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL 8.0
+Community Edition
+<https://dev.mysql.com/doc/refman/8.0/en/mysql-nutshell.html>`__ in addition to
+enterprise-grade features developed by Percona.  For a list of highlighted
+features from both MySQL 8.0 and Percona Server for MySQL 8.0, please see the
+`GA release announcement
+<https://www.percona.com/blog/2018/12/21/announcing-general-availability-of-percona-server-for-mysql-8-0/>`__.
+
+.. note::
+
+   If you are upgrading from 5.7 to 8.0, please ensure that you read the
+   `upgrade guide
+   <https://www.percona.com/doc/percona-server/8.0/upgrading_guide.html>`__ and
+   the document `Changed in Percona Server for MySQL 8.0
+   <https://www.percona.com/doc/percona-server/8.0/changed_in_version.html>`__.
+
+New Features
+================================================================================
+
+- Key rotation for the redo log has been redesigned to use the lsn and block size to calculate the header number.
+  
+
+Bugs Fixed
+================================================================================
+
+- Parallel doublewrite buffer implementation changes the write call to an os_file_write, which returns the error to the caller. Bug fixed :psbug:`5678`.
+
+- Change the innodb-temp-tablespace encryption to OFF during runtime does not un-encrypt the subsequent file-per-table temporary tables.  Bugs fixed :psbug:`5734`.
+
+- In the Redo log encryption, a redesign the key rotation to read the lsn from the header and store the lsn in the encryption structure. The key rotation uses the lsn and the block size to calculate the header number. Bug fixed :psbug:`5565`.
+
+- Turning `ON` the encryption for the system tablespaces generates the encryption key and encrypts system temporary tablespace pages. Turning the encryption `OFF`, all subsequent pages are written to the temporary tablespace without encryption. The tablespace keys are not erased, therefore any encrypted tables can be decrypted. Modifying the innodb_temp_tablespace_encrypt does not affect file-per-table temporary tables. This type of table is encrypted if `ENCRYPTION='y'` at table creation.
+  Bug fixed :psbug:`5736`.
+
+- The redo log encryption fails to error out if the keyring plugin configuration is missing. Bug fixed :psbug:`5476`.
+
+- The rocksdb_large_prefix allows index key prefixes longer than 767 bytes. The :variable:`rocksdb_file_per_table=true` is required. Bug fixed :psbug:`5655`.
+
+- Long shutdowns on a server with over a million tables.
+  Bug fixed :psbug:`5639`.
+
+- `XtraDB` tracks the changed pages. The tracking is controlled by the enabled :variable:`innodb_track_changed_pages`. This process may generate multiple error messages.
+  Bug fixed :psbug:`5541`.
+
+- Large page allocations with `innodb_buffer_pool_chunk_size=4G` causes an incorrect size for the memory segment. Bug fixed
```

Comment:
Upstream bug link too
What is "the memory segment?"

Reply:
Added link - worked with George to rewrite text.